src/pages/AddLiquidityV2/PoolPriceBar.tsx

```diff
@@ -21,6 +21,9 @@ export function PoolPriceBar({
   price?: Price<Currency, Currency>
 }) {
   const theme = useTheme()
+  const canInvertPrice = Boolean(price?.baseCurrency && price.quoteCurrency && !price.baseCurrency.equals(price.quoteCurrency))
```

Comment:
some of these price accessors have ? and some don't. do we need them?

Reply:
the first one makes sense because if there is no price it will return false and won't evaluate the next condition

Reply:
i think it'd be better as `price && price.baseCurrency && price.quoteCurrency && !price.baseCurrency.equals(price.quoteCurrency)`

Reply:
done